meshnets/main.py

```diff
@@ -9,32 +9,37 @@
 
 from meshnets.utils import data_loading
 from meshnets.utils import data_processing
+#from meshnets.utils import data_visualization
 
 FLAGS = flags.FLAGS
-
-flags.DEFINE_string('input_object', 'data/motorBike.obj',
+flags.DEFINE_string('input_object', 'data/blob.vtk',
                     'File path of the object to be visualized')
 
 
 def main(_):
 
-    logging.info('Loading the object from %s', FLAGS.input_object)
-    nodes, cells = data_loading.load_mesh_from_obj(FLAGS.input_object,
-                                                   verbose=True)
+    #data_visualization.plot_data(FLAGS.input_object, verbose=True)
+
+    logging.info('Loading the mesh data from %s', FLAGS.input_object)
+    mesh = data_loading.load_edge_mesh_pv(FLAGS.input_object,
+                                          get_pressure=True,
+                                          verbose=False)
+    nodes, edges, pressure = mesh[0], mesh[1], mesh[2]
```

Comment:
see if this works:

nodes, edges, pressure = mesh[0:2]

Reply:
It does not. This implementation avoids raising a pylint error due to the varying size of the return from our method